### PR TITLE
changed Airport Server and Places Server number.

### PR DESCRIPTION
**Pull request by Arjunica**

### CODE REVIEW

**Comment by Arjunica** [comment]
Screen Shot 2020-10-19 at 2 49 26 PM

File: src/places.h [hunk 1]
```diff
@@ -61,7 +61,7 @@ struct places_ret {
 };
 typedef struct places_ret places_ret;
 
-#define PLACES_SERVER 0x31234567
+#define PLACES_SERVER 0x3892ACAB
```
Comment (galaxyonfire) [inline src/places.h:64]: Undo this, the places server number need not be changed.

**Review by galaxyonfire — DISMISSED** [review]
please also change the program number in the airport.x file to match the number you are using in the .h for clarity.

File: src/places.h (hunk 1 above)
Comment (Arjunica) [reply src/places.h:64]: It did not work before I changed the number

Comment (galaxyonfire) [reply src/places.h:64]: hm, that's too bad. In that case keep it but also change the program number in the places.x file.

**Review by galaxyonfire — APPROVED** [review]

**Pull request merged**